versions/pre/1.18/1.18-pre1/patches/shared/net/minecraft/world/level/biome/BiomeSource.java.patch

```diff
@@ -1,27 +1,17 @@
 --- a/net/minecraft/world/level/biome/BiomeSource.java
 +++ b/net/minecraft/world/level/biome/BiomeSource.java
-@@ -46,12 +46,6 @@
+@@ -46,12 +46,11 @@
     }
  
     private List<List<PlacedFeature>> m_186727_(List<Biome> p_186728_, boolean p_186729_) {
 -      Object2IntMap<PlacedFeature> object2intmap = new Object2IntOpenHashMap<>();
 -      MutableInt mutableint = new MutableInt(0);
```

Comment:
This patch can probably also be removed, as it doesn't seem to matter whether these two objects are before or after the local record.